### ISSUE TITLE
Fatal error: you must specify '--save','--no-save' or '--vanilla'

### ISSUE THREAD

**Issue by soulspirits** [wsdjeg/SpaceVim#3252]
## Expected behavior, english is required
R language REPL process started correctly.
## The reproduce ways from Vim starting (Required!)
Configed spacevim [[layers]] name="lang#r"  reple_command = "r". launch spavim, open .r r language code script. Hit SPC l s i to start REPL process .  Spacevim complaint the following message:

Fatal error: you must specify '--save','--no-save' or '--vanilla' at last
 
## Debug info

Please press <kbd>SPC h I</kbd>, debug info will be put into clipboard, then paste all content below.

## Environment Information

- OS: linux
- vim version: 8.1.2268
- neovim version: -
- SpaceVim version: 1.4.0-dev
- SpaceVim status: 

```
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   vimrc

no changes added to commit (use "git add" and/or "git commit -a")
```

## Output of the `:SPDebugInfo!`

<details><summary> SpaceVim debug information </summary>

### SpaceVim options :

```toml
  auto_disable_touchpad = 1
  autocomplete_method = 'neocomplete'
  autocomplete_parens = 1
  buffer_index_type = 4
  checkinstall = 1
  colorscheme = 'gruvbox'
  colorscheme_bg = 'dark'
  colorscheme_default = 'desert'
  commandline_prompt = '➭'
  custom_color_palette = []
  custom_plugins = [['lilydjwg/colorizer', {'name': 'lilydjwg/colorizer', 'merged': 0}], ['BlakeWilliams/vim-pry', {'name': 'BlakeWilliams/vim-pry'}], ['jalvesaq/Nvim-R', {'name': 'jalvesaq/Nvim-R'}]]
  debug_level = 1
  default_indent = 2
  disabled_plugins = []
  enable_ale = 0
  enable_bepo_layout = 0
  enable_cursorcolumn = 0
  enable_cursorline = 1
  enable_debug = 0
  enable_googlesuggest = 0
  enable_guicolors = 1
  enable_insert_leader = 1
  enable_javacomplete2_py = 0
  enable_key_frequency = 0
  enable_language_specific_leader = 1
  enable_neocomplcache = 0
  enable_neomake = 1
  enable_os_fileformat_icon = 0
  enable_powerline_fonts = 1
  enable_statusline_bfpath = 0
  enable_statusline_mode = 0
  enable_statusline_tag = 1
  enable_tabline_filetype_icon = 1
  enable_tabline_ft_icon = 0
  enable_vimfiler_filetypeicon = 0
  enable_vimfiler_gitstatus = 0
  enable_vimfiler_welcome = 1
  enable_ycm = 0
  error_symbol = '✖'
  expand_tab = 1
  filemanager = 'vimfiler'
  filetree_direction = 'right'
  filetype_icons = {}
  force_global_config = 0
  gitcommit_issue_icon = ''
  gitcommit_pr_icon = ''
  github_username = ''
  guifont = ''
  hiddenfileinfo = 1
  home_files_number = 6
  hosts_url = 'https://raw.githubusercontent.com/racaljk/hosts/master/hosts'
  info_symbol = 'ⓘ'
  keep_server_alive = 1
  language = ''
  leader_guide_default_group_name = ''
  leader_guide_flatten = 1
  leader_guide_hspace = 5
  leader_guide_max_size = 0
  leader_guide_position = 'botright'
  leader_guide_run_map_on_popup = 1
  leader_guide_sort_horizontal = 0
  leader_guide_submode_mappings = {'<C-C>': 'win_close'}
  leader_guide_vertical = 0
  lint_on_save = 1
  lint_on_the_fly = 0
  max_column = 120
  plugin_bundle_dir = '/home/stephen/.cache/vimfiles/'
  plugin_manager = 'dein'
  plugin_manager_processes = 16
  plugin_name = 'Nvim-R'
  project_rooter_automatically = 1
  project_rooter_patterns = ['.git/', '_darcs/', '.hg/', '.bzr/', '.svn/', '.SpaceVim.d/']
  realtime_leader_guide = 1
  relativenumber = 1
  search_tools = ['rg', 'ag', 'pt', 'ack', 'grep', 'findstr']
  sidebar_direction = ''
  sidebar_width = 30
  smartcloseignoreft = ['tagbar', 'vimfiler', 'defx', 'SpaceVimRunner', 'SpaceVimREPL', 'SpaceVimQuickFix', 'HelpDescribe', 'VebuggerShell', 'VebuggerTerminal', 'SpaceVimTabsManager']
  smartcloseignorewin = ['__Tagbar__', 'vimfiler:default']
  snippet_engine = 'neosnippet'
  src_root = 'E:\sources\'
  statusline_inactive_separator = 'arrow'
  statusline_iseparator = 'nil'
  statusline_left_sections = ['winnr', 'filename', 'major mode', 'search count', 'syntax checking', 'minor mode lighters']
  statusline_right_sections = ['fileformat', 'cursorpos', 'percentage']
  statusline_separator = 'arrow'
  statusline_unicode_symbols = 1
  terminal_cursor_shape = 2
  update_retry_cnt = 3
  version = '1.4.0-dev'
  vim_help_language = 'en'
  vimcompatible = 0
  warning_symbol = '⚠'
  wildignore = '*/tmp/*,*.so,*.swp,*.zip,*.class,tags,*.jpg,*.ttf,*.TTF,*.png,*/target/*,.git,.svn,.hg,.DS_Store,*.svg'
  windows_index_type = 3
  windows_leader = 's'
  windows_smartclose = 'q'
```

### SpaceVim layers :

```toml
[[layers]]
  name="autocomplete"
  auto-completion-return-key-behavior='complete'
  auto-completion-tab-key-behavior='smart'
[[layers]]
  name="checkers"
  show_cursor_error=0
  lint_on_the_fly=1
  enable_neomake=1
[[layers]]
  name="format"
[[layers]]
  name="edit"
[[layers]]
  name="ui"
[[layers]]
  name="core"
[[layers]]
  name="core#banner"
[[layers]]
  name="core#statusline"
[[layers]]
  name="core#tabline"
[[layers]]
  name="shell"
  default_position='top'
  default_height=30
[[layers]]
  name="lang#latex"
[[layers]]
  name="fzf"
[[layers]]
  name="lang#ipynb"
[[layers]]
  name="lang#r"
  repl_command='R'
```

### SpaceVim Health checking :

SpaceVim clipboard support check report:
Checking +clipboard:
      SUCCEED!
SpaceVim environment check report:
Current progpath: vim(/usr/bin/vim)
version: 801
OS: linux
[shell, shellcmdflag, shellslash]: ['/bin/bash', '-c', 0]
SpaceVim lua support check report:
Checking +lua:
      SUCCEED!
SpaceVim python support check report:
Checking +python3:
      SUCCEED!
Checking +python:
      SUCCEED!

### SpaceVim runtime log :

```log
[ SpaceVim ] : logger file  does not exists, only log for current process will be shown!
[ SpaceVim ] [17:09:39] [ Info ] Startup with no argv, current dir is used: ~/
[ SpaceVim ] [17:09:39] [ Info ] find local conf: /home/stephen/.SpaceVim.d/init.toml
[ SpaceVim ] [17:09:39] [ Info ] loadding cached local conf: /home/stephen/.cache/SpaceVim/conf/_home_stephen__SpaceVim_d_init_toml.json
[ SpaceVim ] [17:09:39] [ Info ] start to apply config [local]
[ SpaceVim ] [17:09:39] [ Info ] SpaceVim server startup at:/tmp/spacevim_vim_server
[ SpaceVim ] [17:09:39] [ Info ] Start to find root for: 
[ SpaceVim ] [17:09:39] [ Info ] change to root:/home/stephen
[ SpaceVim ] [17:09:39] [ Info ] try to open SpaceVim welcome page
[ SpaceVim ] [17:09:39] [ Info ] change to root:/home/stephen
[ SpaceVim ] [17:09:40] [ Info ] Start to find root for: /home/stephen/demo/demo_emacs_r.r
[ SpaceVim ] [17:09:40] [ Info ] change to root:/home/stephen
[ SpaceVim ] [17:09:46] [ Info ] Start to find root for: /home/stephen/__REPL__
[ SpaceVim ] [17:09:46] [ Info ] change to root:/home/stephen
[ SpaceVim ] [17:09:46] [ Info ] change to root:/home/stephen
[ SpaceVim ] [17:09:54] [ Info ] Start to find root for: /tmp/v6Z7sZ5/4/issue_report.md
[ SpaceVim ] [17:09:54] [ Info ] change to root:/home/stephen
```
</details>

## Screenshots

If you have any screenshots for this issue, please upload here. BTW you can use https://asciinema.org/ for recording video in terminal.

**Comment by jyscao** [comment]
@soulspirits  the PR in #3253 should fix this specific problem you ran into. 

I didn't play around too much with the R repl on Spacevim yet, but the little that I did while dealing with this issue left much to be desired from the REPL experience. For example, I found the fact that every line was evaluated and its output printed needlessly noisy; not sure if there is some way to suppress the output like `;` in Mathematica. 

So let me know if you have any ideas or thoughts on how to improve the R repl on Spacevim.
